### PR TITLE
Added documentation for JVC Projector Binary Sensor

### PR DESCRIPTION
**Pull request by msavazzi**
## Proposed change

Added description of Binary Sensor for JVC Projector integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/108668
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/jvc_projector.markdown [hunk 1]
```diff
@@ -8,6 +8,7 @@ ha_iot_class: Local Polling
 ha_config_flow: true
 ha_codeowners:
   - '@SteveEasley'
+  - '@msavazzi'
 ha_domain: jvc_projector
 ha_platforms:
```
Comment (joostlek) [inline source/_integrations/jvc_projector.markdown:13]: Please add binary sensor to the platforms

File: source/_integrations/jvc_projector.markdown [hunk 2]
```diff
@@ -44,15 +45,9 @@ The JVC Projector remote platform will create a [Remote](/integrations/remote/)
...
-- `mode_1`
-- `mode_2`
-- `mode_3`
-- `lens_ap`
-- `gamma`
-- `color_temp`
-- `natural`
-- `cinema`
-- `anamo`
-- `3d_format`
```
Comment (joostlek) [inline source/_integrations/jvc_projector.markdown]: Why do we remove this?

**Review by joostlek — DISMISSED** [review]

File: source/_integrations/jvc_projector.markdown (hunk 2 above)
Comment (msavazzi) [reply source/_integrations/jvc_projector.markdown]: I should have not removed it :(

File: source/_integrations/jvc_projector.markdown [hunk 3]
```diff
@@ -56,3 +58,8 @@ The JVC Projector remote platform will create a [Remote](/integrations/remote/)
 - `cinema`
 - `anamo`
 - `3d_format`
+
+### Binary sensor
+
+The status reported is:
+- Power on is True when projector is in status "on", "warming"
```
Comment (c0ffeeca7) [inline source/_integrations/jvc_projector.markdown]:
```suggestion
- **Power on** is **True** when the projector is either in status **on** or **warming**.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @msavazzi :+1:

**Pull request merged**